code/lab/lab.cpp

```diff
@@ -407,14 +407,14 @@ void labviewer_render_model(float frametime)
 		auto lab_render_light_save    = Lab_render_without_light;
 		auto lab_debris_override_save = Motion_debris_enabled;
 		auto lab_envmap_override_save = Envmap_override;
-		auto lab_emissive_light_save  = Cmdline_no_emissive;
+		auto lab_emissive_light_save  = Cmdline_emissive;
 
 		if (Lab_selected_mission == "None") {
 			Lab_render_without_light = true;
 			Motion_debris_enabled    = true;
 		}
 
-		Cmdline_no_emissive = !Lab_emissive_light_override;
+		Cmdline_emissive = !Lab_emissive_light_override;
```

Comment:
Shouldn't this line remove the `!`, just like line 150?

Reply:
You are entirely correct.